engineering/principles-and-practices.md

```diff
@@ -38,7 +38,7 @@ Our mission in engineering is two-fold:
 ## Practices
 ### Ship working software often
 
-We use the practice of continuous integration (CI) to integrate all our work into trunk into `master` on a regular basis, as often as possible. A set of automated tests is run both before and after the merge to validate that no regressions are introduced. If these automated tests fail, the team stops what they are doing and someone fixes the problem immediately.
+We use tools like continuous integration (CI) to enable us to merge code frequently to master (multiple times a day). A set of automated tests with very high code coverage (100%) provide assurance that no regressions are introduced. If the tests fail, the team stops what they are doing to fix the problem immediately.
```

Comment:
I agree with most changes introduced by you, and thank you for doing this! This line:

> We use tools like continuous integration (CI) to enable us to merge code frequently to master (multiple times a day).

is the only one I'd push back on because I don't think CI is a tool as much as a concept/practice. I think we're referencing the wikipedia definition of CI in this line. 

> In software engineering, continuous integration (CI) is the practice of merging all developers' working copies to a shared mainline several times a day.

Otherwise, great work!